<launch>

   <!-- <node pkg="tf" type="static_transform_publisher" name="base_scan_base_link_tf" args= "0.5 0.0 0.3 0.0 0.0 0.0  base_link base_scan 10" />-->
    <node pkg="tf" type="static_transform_publisher" name="laser_broadcaster" args="0 0 0.8 0 0 0 base_link laser 10" />
    <node pkg="lidar_obstacle_detect" type="lidar_obstacle_detect_node" name="lidar_obstacle_detect_node" output="screen">
             
             
       <param name = "lidar_topic"              type = "string"   value = "/scan"/>
       <param name = "imu_topic"                type = "string"   value = "/imu/data"/>
    
       <param name = "LIDAR_Obstacle_angle"     type = "int"      value = "45"/>
       <param name = "LIDAR_Obstacle_distance"  type = "double"   value = "5.0"/> 
       <param name = "LIDAR_Rotation_CCW"       type = "bool"     value = "1"/> 
       <param name = "Lane_Width"               type = "double"   value = "0.5"/> 
       <param name = "Robot_Width"              type = "double"   value = "0.4"/> 
       <param name = "Robot_Width_Tolerance"    type = "double"   value = "0.2"/>        
       
       <param name = "number_of_obstacle_point" type = "int"      value = "20"/>
   
   </node>


</launch>

